timescaledb/tutorials/nfl-fantasy-league.md

```diff
@@ -174,19 +174,249 @@ conn.close()
+    '), ranked_positions as ( '\
+    'select v."position", avg(v.avg_yards) as avg_yards_positions '\
+    'from ranked_vals v '\
+    'group by v."position" '\
+    ') '\
+    'select v.*, p.avg_yards_positions from ranked_vals v '\
+    'left join ranked_positions p on v.position = p.position '\
+    'where v.rank <= 3 and v.position != \'null\' and v.position not in (\'K\', \'P\', \'NT\', \'LS\', \'DT\') '\
+    'order by p.avg_yards_positions desc, v.rank asc '
+
+# Import data into a pandas Dataframe from the database using a cursor
+with psycopg2.connect(CONNECTION) as conn:
+        cursor = conn.cursor()
+        cursor.execute(query)
+        data = cursor.fetchall()
+        df = pd.DataFrame(data, columns=['player_id', 'displayname', 'avg_yards', 'position', 'rank', 'avg_yards_positions'])
+        cursor.close()
+
+
+# Create labels, ticks, and define font information
+labels = ['Free Safety','Strong Safety','Corner Back','Wide Receiver','Safety','Middle Linebacker','Inside Linebacker','Outside Linebacker','Tight End','Quarter Back','Linebacker','Defense Back','Running Back','Halfback','Fullback','Defense End']
+x = np.arange(len(labels))
+yticks = range(1,1700, 100)
+font = {'weight': 'heavy',
+        'size': 16,
+        }
+
+# Create matplotlib grouped barchart
+fig, ax = plt.subplots()
+
+# Define three different sets of bars for each rank level
+bar1 = ax.bar(x - .3, df['avg_yards'].loc[df['rank'] == 1], .3, color='royalblue')
+bar2 = ax.bar(x, df['avg_yards'].loc[df['rank'] == 2], 0.3 , color='cornflowerblue')
+bar3 = ax.bar(x + .3, df['avg_yards'].loc[df['rank'] == 3], 0.3 , color='lightsteelblue')
+
+# Specify label and tick information
+ax.set_ylabel('Average Yards Run')
+ax.set_xlabel('Player\'s Position, Ordered by the Average Yards Run of All Players for Each Position', labelpad=20)
+ax.set_title('Top Three Average Yards Run by Players During One Game, Separated by Position', fontdict=font, pad=20)
+ax.set_xticks(x)
+ax.set_yticks(yticks)
+ax.set_xticklabels(labels, rotation=45)
+
+# Set up bars to display the players name 
+ax.bar_label(container=bar1 ,labels=df['displayname'].loc[df['rank'] == 1], padding=3, rotation=90)
+ax.bar_label(container=bar2 ,labels=df['displayname'].loc[df['rank'] == 2], padding=3, rotation=90)
+ax.bar_label(container=bar3 ,labels=df['displayname'].loc[df['rank'] == 3], padding=3, rotation=90)
+
+# Define figure dimensions
+fig.set_figheight(8)
+fig.set_figwidth(14)
+fig.tight_layout()
+
+plt.show()
+```
+
+Your graph should look like this...
+
+<img class="main-content__illustration" src="" alt="Top Three Players by Position"/>
+
+Coming back to the statement earlier, notice that the average yards overall Free Safety players is higher than that of Wide Receivers (This is because of how we ordered the data, noted above). However, individual Wide Receivers run more yards on average per game. Also, notice that Kyle Juszczyk runs far more on average than other Fullback players. 
```

Comment:
```suggestion
Notice that the average yards overall for Free Safety players is higher than that of Wide Receivers (this is because of how we ordered the data). However, individual Wide Receivers run more yards on average per game. Also, notice that Kyle Juszczyk runs far more on average than other Fullback players. 
```